### PR TITLE
Moved xxxUtil files from Slice to respective slice2xxx folders.

### PR DESCRIPTION
**Pull request by InsertCreativityHere**
Moved all the language specific files that were in cpp/src/Slice into their own respective cpp/src/slice2xxx folders.

We used to need them in Slice for Freeze to use, but this is no longer a concern since we've dropped freeze.

### CODE REVIEW

File: python/modules/IcePy/msbuild/icepy.vcxproj.filters [hunk 1]
```diff
@@ -84,19 +84,19 @@
...
+    <ClCompile Include="..\..\..\..\cpp\src\Slice\SliceUtil.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\..\..\cpp\src\Slice\Scanner.cpp">
+    <ClCompile Include="..\..\..\..\cpp\src\Slice\StringLiteralUtil.cpp" />
+    <ClCompile Include="..\..\..\..\cpp\src\slice2py\Python.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\..\..\cpp\src\Slice\SliceUtil.cpp">
+    <ClCompile Include="..\..\..\..\cpp\src\slice2py\PythonUtil.cpp">
```
Comment (externl) [inline python/modules/IcePy/msbuild/icepy.vcxproj.filters]: Is this change correct?

Comment (pepone) [reply python/modules/IcePy/msbuild/icepy.vcxproj.filters]: Seems correct to me, the is a bit confusing here, but PythonUtil comes from slice2py and SliceUtil from cpp\src\Slice

Comment (externl) [reply python/modules/IcePy/msbuild/icepy.vcxproj.filters]: `Python.cpp` -> `Scanner.cpp`?

Comment (externl) [reply python/modules/IcePy/msbuild/icepy.vcxproj.filters]: Or am I just reading it wrong?

Comment (InsertCreativityHere) [reply python/modules/IcePy/msbuild/icepy.vcxproj.filters]: Yeah, the diff is non the clearest, but in the actual file, everything seems correct.

**Pull request merged**